python-packages/smithy-http/requirements.txt

```diff
@@ -0,0 +1,2 @@
+aiohttp>=3.8.6,<3.10.0
+awscrt>=0.15,<1.0
```

Comment:
Is this for pants/testing? Ideally we'd be able to just install `.[aiohttp]` or `.[awscrt]` during testing to avoid specifying these ranges twice.